File: stara/2009/python/main.py
import zad5_a
import zad5_b
import zad5_c
import zad5_d

# zebranie odpowiedzi z podpunktow i zapisanie ich do pliku
result = '\n'.join([zad5_a.answer, zad5_b.answer,
                    zad5_c.answer, zad5_d.answer])
with open('wyniki.txt', mode='w') as f:
    f.write(result)

File: stara/2009/c++/main.cpp
#include "zad5_a.cpp"
#include "zad5_b.cpp"
#include "zad5_c.cpp"
#include "zad5_d.cpp"
#include <fstream>
using namespace std;

int main()
{
    cout << zad5_a() << endl
         << endl
         << zad5_b() << endl
         << endl
         << zad5_c() << endl
         << endl
         << zad5_d() << endl;

    ofstream result("wyniki.txt");

    result << zad5_a() << endl
         << endl
         << zad5_b() << endl
         << endl
         << zad5_c() << endl
         << endl
         << zad5_d() << endl;
}
